tests/testthat/test-independent-testing-meta_split.R

```diff
@@ -0,0 +1,47 @@
+library(dplyr)
+
+test_that("variable 'RACE' checking", {
+ expect_equal(names(meta_dummy() |> meta_split("RACE")), sort(unique(meta_dummy()$data_population$RACE)))
+}) 
+
+
+test_that("variable 'RACE = WHITE' checking in data_population", {
+  x <- meta_dummy() |> meta_split("RACE")
+  z <- meta_dummy()$data_population %>% filter(RACE=="WHITE")
+  expect_equal(length(x$WHITE$data_population$USUBJID), length(z$USUBJID))
```

Comment:
Instead of comparing the total number of subjects, is it better to compare the USUBJID list?